reservation_units/utils/hauki_exporter.py

```diff
@@ -0,0 +1,85 @@
+from urllib.parse import urljoin
+
+from django.conf import settings
+
+from opening_hours.enums import ResourceType
+from opening_hours.errors import HaukiAPIError, HaukiRequestError
+from opening_hours.hauki_request import make_hauki_get_request
+from opening_hours.resources import (
+    Resource,
+    send_resource_to_hauki,
+    update_hauki_resource,
+)
+from reservation_units.models import ReservationUnit
+
+
+class ReservationUnitHaukiExporter:
+    def __init__(self, reservation_unit: ReservationUnit):
+        self.reservation_unit = reservation_unit
+
+    def _get_parent_id(self):
+        """Tries to get reservation_unit.unit hauki resource id from hauki.
+        This is used when hauki_resource_id is not found from reservation unit's unit.
+        """
+        unit = self.reservation_unit.unit
+        if not unit:
+            return None
+        unit_resource_id = (
+            f"{unit.hauki_resource_data_source_id}:{unit.hauki_resource_origin_id}"
+        )
+        url = urljoin(settings.HAUKI_API_URL, "/")
+        url = urljoin(
+            url,
+            f"/v1/resource/{unit_resource_id}",
+        )
```

Comment:
I think this should have the same effect as the two `urljoin`s:
```python
url = urljoin(
    settings.HAUKI_API_URL,
    f"/v1/resource/{unit_resource_id}",
)
```

Reply:
True it should